week-2/14-align-self/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Flexbox</title>
+  <link rel="stylesheet" href="/css/normalize.css">
+  <link rel="stylesheet" href="/css/week2.css">
+	<link rel="stylesheet" href="/css/week2-flexbox.css">
+  <link rel="stylesheet" href="flexbox.css">
+  <link rel="stylesheet" href="./ignore_this_folder/custom.css">
+</head>
+
+<body>
+  <div class="site-wrapper site-wrapper">
+    <div class="site-header">
+      <h1 class="site-header__title">First</h1>
+    </div>
+    <ul class="countries countries--tall countries--first">
+			<li class="country" id="brazil">Brazil</li>
+			<li class="country" id="croatia">Croatia</li>
+			<li class="country" id="ethiopia">Ethiopia</li>
+			<li class="country" id="laos">Laos</li>
+			<li class="country" id="uganda">Uganda</li>
+    </ul>
+    <p>⬇️⬇️ should look like this ⬇️⬇️</p>
```

Comment:
An image for the separate results if we go that way.

![align-result](https://user-images.githubusercontent.com/2306629/50686215-44bae480-1014-11e9-8392-5151d89e1734.png)